### PR TITLE
Docs migration

### PR DESCRIPTION
**Pull request by ramonguiu**
## Description

Point users to the docs site for documentation and make some updates to the content in the README

### CODE REVIEW

**Review by paulfantom — CHANGES_REQUESTED** [review]
Could you run:
```
go install -a github.com/bwplotka/mdox@latest
make docs
```

and commit changes?

**Comment by ramonguiu** [comment]
> Could you run:
> 
> ```
> go install -a github.com/bwplotka/mdox@latest
> make docs
> ```

@paulfantom I get this error

````
find: illegal option -- t
usage: find [-H | -L | -P] [-EXdsx] [-f path] path ... [expression]
       find [-H | -L | -P] [-EXdsx] -f path [path ...] [expression]
>> formatting and local/remote links
mdox fmt --soft-wraps -l --links.validate.config-file=
make: mdox: No such file or directory
make: *** [docs] Error 1
````

**Comment by paulfantom** [comment]
Seems like `mdox` wasn't installed before running `make docs`. Ensure you have at least golang 1.17 and run `go install -a github.com/bwplotka/mdox@latest`. Also check if `$GOBIN` is part of your `$PATH`.

---
Side note: we might want to invest some time into figuring out better management of development tools like `mdox` across developer environments.

File: README.md [hunk 1]
```diff
@@ -52,194 +60,12 @@ data migration tool**, that migrates data from one storage system to another, le
...
-      - source_labels: [__name__]
-        regex: 'go_.*'
-        action: drop
-```
+# Documentation
 
-Additional information about setting up relabel configs, the
-`source_labels` field, and the possible actions can be found in the
-[Prometheus Docs](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write).
+Complete user documentation is available at https://docs.timescale.com/promscale/latest/
```
Comment (cevian) [inline README.md:68]: can we make this a link?
```suggestion
Complete user documentation is available at [https://docs.timescale.com/promscale/latest/](https://docs.timescale.com/promscale/latest/)
```

File: docs/alerting.md [hunk 2]
```diff
@@ -1,38 +1,3 @@
...
-```
-alerting:
-  alertmanagers:
-  - static_configs:
-    - targets:
-      - localhost:9093
-```
-
-More details on alerting rules can be found [here](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/).
+The content in this page has been moved to https://docs.timescale.com/promscale/latest/alert/
```
Comment (cevian) [inline docs/alerting.md:3]: let's make all of these links as well?

**Review by cevian — APPROVED** [review]

**Review by paulfantom — APPROVED** [review]

**Pull request merged**